normalize/main.scss

```diff
@@ -1,14 +1,18 @@
 @mixin nUiNormalize {
 
 	/**
-	* Prevent adjustments of font size after orientation changes in IE and iOS.
+	* Apply font smoothing;
+	* Prevent adjustments of font size after orientation changes in IE and iOS
 	*/
 
 	html {
+		// scss-lint:disable VendorPrefix
+		-moz-osx-font-smoothing: antialiased;
```

Comment:
I don't think such a value exists, do you mean `grayscale`?

Reply:
Urgh, yep, thanks!

New favourite facepalm:
![](https://media.giphy.com/media/xT1XGyUR9PSfwfUeAw/giphy.gif)

Reply:
that is incredible